libs/velo-external-db-core/src/service/schema.ts

```diff
@@ -33,10 +33,11 @@ export default class SchemaService {
             await this.storage.list() : 
             await Promise.all(collectionIds.map((collectionName: string) => this.schemaInformation.schemaFor(collectionName)))
 
-        return { collections: collections.map(this.formatCollection.bind(this)) }
+                return { collections: collections.map(this.formatCollection.bind(this)) }
     }
 
-    async create(collection: collectionSpi.Collection): Promise<collectionSpi.CreateCollectionResponse> {                
+    async create(collection: collectionSpi.Collection): Promise<collectionSpi.CreateCollectionResponse> {   
+        await this.validateFields(collection.fields)
```

Comment:
if its not async you can remove await :) 
thats the last comment promise - you can ignore as well, bye